src/components/Card/Card.tsx

```diff
@@ -1,48 +1,48 @@
-import { Link } from 'react-router-dom'
-import './Card.css';
+import { Link } from 'react-router-dom';
+import '../Card/Card.css';
 
-interface CardProps {
+interface Card {
+  id: string
+  shortUrl: string
   title: string
   image: string
-  id: string
   sentiment: number
-  topic: string
   updateUserSentiment: (userSentiment: number) => void
   storeArticle: (id: string) => void
+  topic: string
 }
 
-const Card = ({ title, image, id, sentiment, topic, updateUserSentiment, storeArticle }: CardProps): JSX.Element => {
+const Card = ({ title, image, id, shortUrl, sentiment, topic, updateUserSentiment, storeArticle }: Card): JSX.Element => {
 
-  const handleClick = (): void => {
-    storeArticle(id)
+  const handleClick = () => {
     updateUserSentiment(sentiment)
+    storeArticle(id)
   }
 
   return (
-    <div className="article-boundary">
-      <div className="card-container">
-        <article className="news-card cy-article-card">
-          <Link
-            to={`/feed/${id}`}
-            onClick={ handleClick }
-            className="cy-article-link"
-          >
-          <div className="article-image-container">
-            {sentiment >= 7 && <h3 className="article-sentiment green">{sentiment}</h3>}
-            {sentiment <= 3 && <h3 className="article-sentiment red">{sentiment}</h3>}
-            {(sentiment >= 4 && sentiment <= 6) && <h3 className="article-sentiment blue">{sentiment}</h3>}
-            <img className="article-image cy-article-image" src={image} alt={title} />
-          </div>
-          <div className="topic-container">
-            <p className="topic-text">
-              {topic}
-            </p>
-          </div>
-          <h2 className="article-title cy-article-title">{title}</h2>
-          </Link>
-          <div className="divider"></div>
-        </article>
-      </div>
+    <div className="card-container">
+      <article className="news-card cy-article-card">
+        <a
+          onClick={ handleClick }
+          className="cy-article-link"
+          href={ shortUrl }
+          target='_blank'
+        >
+        <div className="article-image-container">
+          {sentiment >= 7 && <h3 className="article-sentiment green">{sentiment}</h3>}
+          {sentiment <= 3 && <h3 className="article-sentiment red">{sentiment}</h3>}
+          {(sentiment >= 4 && sentiment <= 6) && <h3 className="article-sentiment blue">{sentiment}</h3>}
+          <img className="article-image cy-article-image" src={image} alt={title} />
+        </div>
+        <div className="topic-container">
+          <p className="topic-text">
+            {topic}
+          </p>
+        </div>
+        <h2 className="article-title cy-article-title">{title}</h2>
+        </a>
```

Comment:
What do you all think about adding the abstract to the article cards since we are not displaying the article anymore?